docs/source/usage/data.rst

```diff
@@ -56,6 +57,20 @@ data set in a directory called `stanford_hardi`::
                 ├── sub-01_ses-01_dwi.bvec
                 └── sub-01_ses-01_dwi.nii.gz
 
+See :ref:`examples/plot_bids_layout` for a more extensive example.
+
+
+Preprocessing
+-------------
+pyAFQ does not perform preprocessing. Instead, pyAFQ expects the outputs of
+a preprocessing pipeline in BIDS format. In the above example, the dataset
+was preprocessed using the `VISTASOFT <https://github.com/vistalab/vistasoft>`_ software.
+The outputs of VISTASOFT are stored in its own pipeline folder,
+`derivatives/vistasoft`. Other examples may use other common preprocessing
```

Comment:
clarify that it doesnt have to be derivatives/vistasoft, it could be derivatives/custompipeline